DiceKeys/Views/Screens/CustomeDiceDemo/BlankView.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  BlankView.swift
+//  DiceKeys
+//
+//  Created by Kevin Shah on 16/01/21.
+//
+
+import SwiftUI
+
+/// BlankViewModel
+class BlankViewModel: ObservableObject {
+    
+    @Published var isSelected: Bool
```

Comment:
Rather than have 25 booleans for which die is selected, it is better to have a single optional Int (Int?) for which selectedDieIndex, and then have a calculated state:

var isSelected: Bool { thisDieIndex == selectedDieIndex }

that can go directly into the view since the state (model) data is maintained externally.

Reply:
BlankView file is remove from project.